chef_master/source/chef_overview.rst

```diff
@@ -15,10 +15,12 @@ This diagram shows how you develop, test, and deploy your Chef code.
 
 .. end_tag
 
-* The **Chef DK workstation** is the location where users interact with Chef. On the workstation users author and test `cookbooks </cookbooks.html>`__ using tools such as `Test Kitchen </kitchen.html>`__ and interact with the Chef server using the `knife </knife.html>`__ and `chef </ctl_chef.html>`__ command line tools.
+* The **Chef Workstation** is the location where users interact with Chef. On the workstation users author and test `cookbooks </cookbooks.html>`__ using tools such as `Test Kitchen </kitchen.html>`__ and interact with the Chef server using the `knife </knife.html>`__ and `chef </ctl_chef.html>`__ command line tools.
```

Comment:
Can we remove "The"

Reply:
Also, I haven't seen this kind of wording before "On the workstation" ... hmm.

Reply:
Right? Well, this would be one of the oldest docs, so it may have hung around for this long.

Reply:
done

Reply:
On the workstation... That's the problem with calling this product "Chef Workstation" because the workstation is a familiar part of the sysadmin physical environment--it is where they install DK/Workstation and interact with nodes and servers.